src/storage/src/sst/parquet.rs

```diff
@@ -282,6 +297,121 @@ impl<'a> ParquetReader<'a> {
 
         ChunkStream::new(adapter, Box::pin(chunk_stream))
     }
+
+    /// Builds time range row filter.
+    fn build_time_range_row_filter(&self, schema_desc: &SchemaDescriptor) -> Option<RowFilter> {
+        let ts_col_idx = self
+            .projected_schema
+            .schema_to_read()
+            .schema()
+            .timestamp_index()?;
+        let ts_col = self
+            .projected_schema
+            .schema_to_read()
+            .schema()
+            .timestamp_column()?;
+
+        let ts_col_unit = match &ts_col.data_type {
+            ConcreteDataType::Int64(_) => TimeUnit::Millisecond,
+            ConcreteDataType::Timestamp(ts_type) => ts_type.unit(),
+            _ => unreachable!(),
+        };
+
+        // build lower and upper bound according to time range and timestamp column data type.
+        let lower = self
+            .time_range
+            .start()
+            .and_then(|s| s.convert_to(ts_col_unit))
+            .map(|t| t.value())
+            .unwrap_or(i64::MIN);
```

Comment:
Maybe we should read data from the special time window (i64::MAX) instead of using `(min, max)` as the time range, especially during compaction.